packages/web-components/src/components/breadcrumb/breadcrumb.ts

```diff
@@ -66,6 +66,18 @@ class CDSBreadcrumb extends LitElement {
     `;
   }
 
+  updated(changedProperties) {
+    if (changedProperties.has('size')) {
+      const items = document
+        .querySelector(`${prefix}-breadcrumb`)
+        ?.querySelectorAll(`${prefix}-breadcrumb-item`);
```

Comment:
```suggestion
      const items = this.querySelectorAll(`${prefix}-breadcrumb-item`);
```

You'll want to scope the querySelector to the current instance of breadcrumb instead of `document` otherwise if there are other instances of breadcrumb on the page, it will affect those other instances as well